package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@d-i-t-a/reader",
-  "version": "1.8.5",
+  "version": "2.0.0",
```

Comment:
I think actually we should make this 2.0.0-beta.1 and keep publishing changes to the beta until we have settled a bit. I think there may be a few more breaking changes we want to make in addition to the new API, for example if we implement strictNullChecks or change some of the defaults, etc. I think it's safest to make it a beta release for a little while so we can add more breaking changes if need be without annoying the people using the stable version. That way they just upgrade a single time when we release 2.0.0

Reply:
We discussed this in slack that we are going to use 1.9.0 for this release

Reply:
yes i will make sure to change this to 1.9

Reply:
the 2.0.0 then will anyay stay in alpha and then beta for a while.